tests/end_to_end/candid_rpc/functional_syntax/notify_raw/test/test.ts

```diff
@@ -5,16 +5,18 @@ import { createActor as createActorCanister1 } from './dfx_generated/canister1';
 import { createActor as createActorCanister2 } from './dfx_generated/canister2';
 import { getTests } from './tests';
 
-const canister1 = createActorCanister1(getCanisterId('canister1'), {
+const canister1Name = 'canister1';
+const canister1 = createActorCanister1(getCanisterId(canister1Name), {
     agentOptions: {
         host: 'http://127.0.0.1:8000'
     }
 });
 
-const canister2 = createActorCanister2(getCanisterId('canister2'), {
+const canister2Name = 'canister2';
+const canister2 = createActorCanister2(getCanisterId(canister2Name), {
     agentOptions: {
         host: 'http://127.0.0.1:8000'
     }
 });
 
-runTests(getTests(canister1, canister2));
+runTests(getTests(canister1, canister2), canister1Name);
```

Comment:
Just one?

Reply:
how about you get off my back about all of these guys?

Reply:
done